pkg/api/handlers/compat/containers.go

```diff
@@ -403,6 +403,32 @@ func LibpodToContainerJSON(l *libpod.Container, sz bool) (*types.ContainerJSON,
 		state.Status = define.ContainerStateCreated.String()
 	}
 
+	state.Health.Status = inspect.State.Healthcheck.Status
+
+	state.Health.FailingStreak = inspect.State.Healthcheck.FailingStreak
+
+	//item := define.HealthCheckLog{}
+
+	var item (*types.HealthcheckResult)
+
+	log := inspect.State.Healthcheck.Log
+
+	for ind := range log {
+		s, err := time.Parse("2019-12-22T10:59:05.6385933Z", log[ind].Start)
```

Comment:
I'm ~90% sure this is RFC3339Nano which should already be defined in the Go time package

Reply:
yep, you're right. Any idea why on APIv2 test these lines parsing the time (I think) seem to be causing an issue? The test spits out: 
` not ok 726 [44-mounts] GET containers/7d2916da153cfbf7866eb530a68f9d499f4896810a068410e5674ae8263dd3cb/json : status
         #  expected: 200
         #    actual: 500
         #  response: {"cause":"runtime error: invalid memory address or nil pointer dereference","message":"runtime error: invalid memory address or nil pointer dereference","response":500}`

Reply:
I think you need a line number - is the server spitting anything out during this? I'd try and run the test locally and verify output of the server.